docs: add reference to example dir (#80)
amrbashir/tauri-plugin-vibrancy: README.md

apply_blur(&window, Some((18, 18, 18, 125))).expect("Unsupported platform! 'apply_blur' is only supported on Windows");
```

> For a more complete example of usage with [tauri](https://tauri.app/), view the ['example'](/example) directory of this repository

Reviewer: ```suggestion
For a more complete example of usage with [tauri](https://tauri.app/), see [`examples/tauri`](https://github.com/tauri-apps/window-vibrancy/tree/dev/examples/tauri).
```
